docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: PROCEDURE
+
+[id="hermetic-builds_{context}"]
+= Enabling hermetic builds
+
+Hermetic builds refer to a controlled build process where all required resources and dependencies are declared and prefetched within a build system. This strict control ensures that the build system has complete visibility and control over the dependencies needed for the build, including the ability to accurately collect, inquire, analyze, and manage all dependencies during the build process. By restricting external dependencies and implementing a centralized management approach, hermetic builds provide a predictable and reliable build environment.
+
+IMPORTANT: If you do not vendor all the required dependencies in a git repository while isolated from the network, your build may fail. 
+
+Hermetic builds also focus on establishing a build pipeline that captures and maintains a precise record of the build’s origin, dependencies, and modifications. This emphasis on provenance allows for better traceability, reproducibility, and auditability of the build process. By having comprehensive visibility into the entire pipeline, hermetic builds enable the identification and resolution of issues, facilitate collaboration, and support long-term maintenance of the builds.
```

Comment:
I am not sure what you are trying to say in these sentences. Don't we already define why provenance is important to us when discussing SLSA? Can we just link over to that to give more background on it? We can briefly introduce the connection here before linking, but defer to the other page for details.